File: python/apps/examples/hello_world.py
from apache_beam import Pipeline, Create
from apache_beam import Map

from apps.examples.examples import Examples
from apps.helpers import print_msg
from apps.schemas import GenericMessage


class HelloWorld(Examples):
    """ """
    def build(self, p: Pipeline):
        (p
         | Create([GenericMessage(message="Hello world")])
        | 'Print' >> Map(print_msg) )

File: java/src/main/java/app/examples/HelloWorld.java
package app.examples;

import app.helpers.fns.PrintFn;
import org.apache.beam.sdk.Pipeline;
import org.apache.beam.sdk.transforms.Create;
import org.apache.beam.sdk.transforms.MapElements;
import org.apache.beam.sdk.values.PCollection;

public class HelloWorld implements Examples{

    public void build(Pipeline p) {
        PCollection<String> strings = p.apply(Create.of("Hello World!"));
        strings.apply(MapElements.via(new PrintFn()));
    }
}
